site/content/en/docs/concepts/admission_check.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Admission Check"
+date: 2023-10-05
+weight: 6
+description: >
+  A mechanism allowing custom internal or external components to control the timing of
+  workloads admission.
+---
```

Comment:
Let's add a paragraph here that explains the high level idea:

AdmissionChecks are a mechanism that allows Kueue to consider additional criteria before admitting a Workload. When a ClusterQueue has AdmissionChecks configured, each of the checks has to provide a
positive signal to the Workload before it can be admitted (link to the admission definition).
You can configure the mechanism using the following components:

Reply:
done